aiohttp/formdata.py

```diff
@@ -117,8 +116,8 @@ def _gen_form_urlencoded(self) -> payload.BytesPayload:
 
     def _gen_form_data(self) -> multipart.MultipartWriter:
         """Encode a list of fields using the multipart/form-data MIME format"""
-        if self._is_processed:
-            raise RuntimeError("Form data has been processed already")
+        if not self._fields:
+            return self._writer
```

Comment:
This doesn't appear to save much, one call to .clear().
```suggestion
```

Reply:
> This doesn't appear to save much, one call to .clear().

agree with this